plugins/es-analysis-common/src/test/java/io/crate/analysis/common/CommonAnalyzerITest.java

```diff
@@ -33,6 +34,7 @@
 import io.crate.lucene.DisableGraphAttributeImpl;
 import io.crate.testing.TestingHelpers;
 
+@IntegTestCase.ClusterScope(minNumDataNodes = 2)
```

Comment:
Is this important for the new scenarios or can we remove this again?

Reply:
Given that the mechanism is the same for one node only, I could revert, but I'd feel more safe to keep it to be able to catch potential future regressions, WDYT?